index.js

```diff
@@ -0,0 +1,45 @@
+let image = document.querySelector(".profile-pic");
+
+image.onclick = function () {
+  alert("Not available!");
+}
+
+let myHeaders = new Headers();
+myHeaders.append("Origin", "https://localhost.com");
+
+let requestOptions = {
+  method: 'GET',
+  headers: myHeaders,
+  redirect: 'follow'
+};
+let gallery = document.querySelector(".picture-grid");
+
+
+let arrayPromise = fetch("https://image-gallery-pjks.s3.ca-central-1.amazonaws.com/data.json", requestOptions)
+  .then(res => res.json())
+  .then(res => {
+    console.log(res)
+    return res;
+  });
```

Comment:
You can just chain promises so instead of making a separate variable below so in this cause it would look like this

```javascript
let arrayPromise = fetch("https://image-gallery-pjks.s3.ca-central-1.amazonaws.com/data.json", requestOptions)
  .then(res => res.json())
  .then(response => parseData(response)); //I would make a separate function if possible

function parseDate(response) {
  //The code below
}
```